android-client-sdk/src/test/java/com/devcycle/sdk/android/api/DVCClientTests.kt

```diff
@@ -383,38 +383,6 @@ class DVCClientTests {
         client.close()
     }
 
-    @Test
-    fun `variable calls back when variable value changes`() {
-        val config = generateConfig("activate-flag", "Flag activated!", Variable.TypeEnum.STRING)
-
-        mockWebServer.enqueue(MockResponse().setResponseCode(200).setBody(objectMapper.writeValueAsString(config)))
-
-        val client = createClient("pretend-its-a-real-sdk-key", mockWebServer.url("/").toString())
-
-        try {
-            val variable = client.variable("activate-flag", "Not activated")
-            variable.onUpdate(object: DVCCallback<Variable<String>> {
```

Comment:
@VicV this approach for onUpdate seems to be deprecated in favour of using a plain function (see test below), removing this has all tests passing. I believe this is related to the bump to java 18

Reply:
@laurawarr I don't have any context around this test / onupdate function im just a man that plays around with gradle you might want a diff reviewer on that